pkg/apis/objectbucket.io/v1alpha1/register.go

```diff
@@ -14,6 +14,10 @@ import (
 // SchemeGroupVersion is group version used to register these objects
 var SchemeGroupVersion = schema.GroupVersion{Group: objectbucketio.GroupName, Version: "v1alpha1"}
 
+var OBCgroupVersion = SchemeGroupVersion.Group + "/" + SchemeGroupVersion.Version
+
+var OBCgvk = schema.GroupVersionKind{Group: SchemeGroupVersion.Group, Version: SchemeGroupVersion.Version, Kind: "ObjectBucketClaim"}
```

Comment:
per bjeans convo,

drop this line.  add

```
const Version = "v1alpha1"

func GroupKindVersion(kind string) schema.GroupVersionKind {
	return SchemeGroupVersion.WithKind(kind)
}
```

Add to objectbucket_types.go

```
const ObjectBucketKind = "ObjectBucket"

func ObjectBucketGVK() schema.GroupVersionKind {
	return GroupKindVersion(ObjectBucketKind)
}
```

add to objectbucketclaim_types.go

```
const ObjectBucketClaimKind = "ObjectBucketClaim"

func ObjectBucketClaimGVK() schema.GroupVersionKind{
	return GroupKindVersion(ObjectBucketClaimKind)
}
```